tests/dataimportcron_test.go

```diff
@@ -0,0 +1,219 @@
+package tests_test
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/api/batch/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+	cdiclientset "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned"
+	"kubevirt.io/containerized-data-importer/pkg/controller"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+const (
+	dataImportCronTimeout = 90 * time.Second
+)
+
+var _ = Describe("DataImportCron", func() {
+	const (
+		scheduleEveryMinute = "* * * * *"
+		scheduleOnceAYear   = "0 0 1 1 *"
+	)
+	var (
+		f                   = framework.NewFramework(namespacePrefix)
+		registryPullNode    = cdiv1.RegistryPullNode
+		trustedRegistryURL  = func() string { return fmt.Sprintf(utils.TrustedRegistryURL, f.DockerPrefix, f.DockerTag) }
+		externalRegistryURL = "docker://quay.io/kubevirt/fedora-cloud-registry-disk-demo:latest"
+		cron                *cdiv1.DataImportCron
+		err                 error
+	)
+
+	AfterEach(func() {
+		if cron != nil {
+			By("Delete cron")
+			err = DeleteDataImportCron(f.CdiClient, cron.Namespace, cron.Name)
```

Comment:
make sure cluster clean script removes any finalizers as well, just in case.  Look here:

https://github.com/kubevirt/containerized-data-importer/blob/main/cluster-sync/clean.sh#L23-L35

Reply:
done